components/create/Review.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import Chip from '@mui/material/Chip';
+import Stack from '@mui/material/Stack';
+import CreatePortfolioContext from 'components/create/CreatePorfolioContext';
+import { Typography } from '@mui/material';
+import { useState, useContext, useRef, RefObject } from 'react';
+import { Divider } from '@mui/material';
+
+const Review = () => {
+  const { title, name, headshot, about, work, skills, projects, contact } =
+    useContext(CreatePortfolioContext);
```

Comment:
```
const portfolio = {
  content: { title, name, headshot... }
} as IPortfolio
```

so that code on view page is the same as on the review page :)

Reply:
Done.

Reply:
fixed